simple_certmanager/models.py

```diff
@@ -87,6 +79,16 @@ def save(self, *args, **kwargs):
                 country=self.country_name,
                 email=self.email_address,
             )
+        else:
+            if not self.public_certificate:
+                self.csr = generate_csr(
+                    key_pem=self.private_key,
+                    common_name=self.common_name,
+                    organization_name=self.organization_name,
+                    locality=self.locality_name,
+                    state_or_province=self.state_or_province_name,
+                    country=self.country_name,
+                )
```

Comment:
@CharString I'm confused - I thought the CSR would be generated on the fly and not stored in the DB anymore?

Reply:
Yeah, this can be moved to a
```
@property
def csr(self) -> CSR:
    generate_csr(
        key_pem=self.private_key,
        common_name=self.common_name,
        organization_name=self.organization_name,
        locality=self.locality_name,
        state_or_province=self.state_or_province_name,
        country=self.country_name,
    )
```